nbconvert/nbconvertapp.py

```diff
@@ -93,7 +93,14 @@ def validate(self, obj, value):
         },
         """Run nbconvert in place, overwriting the existing notebook (only 
         relevant when converting to notebook format)"""
+        ),
+    'tiemout' : (
```

Comment:
I wouldn't expect this to be a flag (which means that you give it as `--timeout` with no argument). If we're adding it to the command line interface, I'd expect it to be an option, so you give e.g. `--timeout 20`.